SSW.Rules.AzFuncs/Functions/Bookmarks/GetBookmarkStatusFunction.cs

```diff
@@ -42,25 +40,21 @@ public async Task<HttpResponseData> Run(
         }
 
         _logger.LogInformation("Checking for bookmark on rule: {0} and user: {1}", ruleGuid, userId);
-        var bookmarks =
-            await dbContext.Bookmarks.Query(q => q.Where(w => w.RuleGuid == ruleGuid && w.UserId == userId));
+        var bookmarks = await dbContext
+            .Bookmarks
+            .Query(q => q.Where(w => w.RuleGuid == ruleGuid && w.UserId == userId));
         
-        if (bookmarks.Any())
+        var bookmarkStatus = bookmarks.Any();
+        if (bookmarkStatus)
```

Comment:
This should be reversed right? We want to log when there are no bookmarks found

```suggestion
        if (!bookmarkStatus)
```